assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -123,32 +126,32 @@ export class CounterFrontend extends Component {
             </header>
           }
           {description && !isEditing &&
-            <p className="page-section-description" dangerouslySetInnerHTML={{ __html: description }} />
+            <p className="page-section-description" dangerouslySetInnerHTML={{__html: description}} />
           }
           <div className="content-counter">
             {(style === 'bar' || style === 'en-forms-bar') &&
               <div className="progress-container">
-                <div className={`progress-bar ${style === 'en-forms-bar' ? 'enform-progress-bar' : ''}`} style={{ width: `calc(${percent}% + 20px)` }} />
+                <div className={`progress-bar ${style === 'en-forms-bar' ? 'enform-progress-bar' : ''}`} style={{width: `calc(${percent}% + 20px)`}} />
               </div>
             }
             {style === 'arc' &&
               <svg className="progress-arc" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 24 14">
                 <path className="background" d="M 2 12 A 1 1 0 1 1 22 12" />
                 <path className="foreground" d="M 2 12 A 1 1 0 1 1 22 12"
                   strokeDasharray={arcLength}
-                  strokeDashoffset={`${(1 - percent / 100) * arcLength}`} />
+                  strokeDashoffset={`${((1 - percent) / 100) * arcLength}`} />
```

Comment:
Are you sure this isn't supposed to be the other way around (`1 - (percent / 100)`)?

Reply:
I have no idea, but I will change it.